### PR TITLE
WIP: Add New Address Screen

### PR DESCRIPTION
**Pull request by Josetf2396**
image

### CODE REVIEW

File: mobile/src/screens/addNewAddress/addNewAddress.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React, {FC} from 'react';
+import {EAuthNavigationRoutes} from '../../navigation/authNavigation/AuthNavigation.type';
+import {TaddNewAddressNavigation} from './addNewAddress.type';
+import {AddNewAddressScreenView} from './addNewAddress.view';
+
+export const AddNewAddressScreen: FC<TaddNewAddressNavigation> = ({navigation}) => {
+  return (
+    <AddNewAddressScreenView
+      name='Eljad Eendaz'
```
Comment (lasharela) [inline mobile/src/screens/addNewAddress/addNewAddress.tsx:9]: Needs to be props for main component.

**Review by lasharela — CHANGES_REQUESTED** [review]

**Review by raynesmike — APPROVED** [review]

File: mobile/src/screens/addNewAddress/addNewAddress.style.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        padding: size.sm,
+        fontfamily: font.bold
+    },
+    title: {
+        fontSize: size.lg,
+        fontFamily: font.semiBold,
+        color: color.black,
+    },
+    button: {
+        padding: size.rg,
```
Comment (Josetf2396) [inline mobile/src/screens/addNewAddress/addNewAddress.style.ts:21]: remove padding

File: mobile/src/screens/addNewAddress/addNewAddress.view.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      <Header
+        leftIconName="ChevronLeft"
+        title="Add new address"
+        iconWidth={size.rg}
+        iconHeight={size.rg}
+        containerStyle={styles.header}
+        leftPress={goBack}
+      />
+      <View style={styles.container}>
+        <Text style={styles.text}>Full name</Text>
```
Comment (lasharela) [inline mobile/src/screens/addNewAddress/addNewAddress.view.tsx:30]: <Section>Wrapp rest here</Section>

Comment (lasharela) [reply mobile/src/screens/addNewAddress/addNewAddress.view.tsx:30]: == View == 
topSize
topReset

bottomSize
bottomReset

verticalSize
verticalReset

**Review by lasharela — CHANGES_REQUESTED** [review]

File: mobile/src/screens/addNewAddress/addNewAddress.style.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        padding: size.sm,
+        fontfamily: font.bold
+    },
+    title: {
+        fontSize: size.lg,
+        fontFamily: font.semiBold,
+        color: color.black,
+    },
+    button: {
+        shadowColor: color.black,
```
Comment (lasharela) [inline mobile/src/screens/addNewAddress/addNewAddress.style.ts:27]: on iOS this won't work

File: mobile/src/screens/addNewAddress/addNewAddress.type.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+export type TLoginScreenProps = StackScreenProps<TAuthNavigationRoutes, EAuthNavigationRoutes.LOGIN>;
+
+export type TaddNewAddressViewProps = {
+    name?: string;
+    phoneNumber?: string;
+    state?: string;
+    city?: string;
+    street?: string;
+    saveButton?: () => void;
+    goBack?: () => void;
```
Comment (lasharela) [inline mobile/src/screens/addNewAddress/addNewAddress.type.ts:13]: () => void  
Function 

saveButton?: () => void;

const myMethod = {()=> {do something)}
<Component
saveButton={myMethod} />

File: mobile/src/screens/addNewAddress/addNewAddress.view.tsx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+        <Header
+          leftIconName="ChevronLeft"
+          title="Add new address"
+          iconWidth={size.rg}
+          iconHeight={size.rg}
+          containerStyle={styles.header}
+          leftPress={goBack}
+        />
+      }>
+      <View style={styles.container}>
```
Comment (lasharela) [inline mobile/src/screens/addNewAddress/addNewAddress.view.tsx:29]: <View style={styles.container}>

**Review by lasharela — APPROVED** [review]

**Review by lasharela — APPROVED** [review]

**Review by Dulaya — APPROVED** [review]

**Review by mshago — APPROVED** [review]

**Review by aahenry51 — APPROVED** [review]

**Pull request merged**